Reference/conformance/conformance.pb.swift

```diff
@@ -163,7 +163,13 @@ struct Conformance_ConformanceRequest: SwiftProtobuf.Message {
   /// other serializer methods are defined in the SwiftProtobuf library. See the
   /// `Message` and `Message+*Additions` files.
   func traverse<V: SwiftProtobuf.Visitor>(visitor: inout V) throws {
-    try self.payload?.traverse(visitor: &visitor)
+    switch self.payload {
```

Comment:
Personally, I prefer the "if let x ... switch x" pattern here because:   We have no handling for the `.none` case; and It doesn't visually repeat the optional unwrap on every case.  But I don't feel that strongly about it.

If you do leave it this way, I suggest changing the `default: break` to `case .none: break`   The former breaks the compiler's exhaustiveness checking, hiding possible bugs.